libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -478,14 +456,8 @@ Vector2f* AC_Fence::get_boundary_points(uint16_t& num_points) const
     return &_boundary[1];
 }
 
-/// returns true if we've breached the polygon boundary.  simple passthrough to underlying _poly_loader object
-bool AC_Fence::boundary_breached(const Vector2f& location, uint16_t num_points, const Vector2f* points) const
-{
-    return _poly_loader.boundary_breached(location, num_points, points);
-}
-
 /// handler for polygon fence messages with GCS
-void AC_Fence::handle_msg(GCS_MAVLINK &link, mavlink_message_t* msg)
+void AC_PolyFence_loader::handle_msg(GCS_MAVLINK &link, mavlink_message_t* msg)
```

Comment:
I'm sure the intention is to move this implementation to the AC_PolygFence_loader.cpp file but just in case this was forgotten.